tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/HttpExceptionMiddleware.cs

```diff
@@ -79,6 +80,8 @@ public async Task Invoke(HttpContext context)
                 response.Headers.Add("x-request-exception", "true");
                 response.Headers.Add("x-request-exception-error", Convert.ToBase64String(Encoding.UTF8.GetBytes(e.Message)));
 
+                DebugLogger.LogError(unexpectedStatusCode.ToString() + Environment.NewLine + e.Message + Environment.NewLine + e.StackTrace);
```

Comment:
Would it make more sense to put this logic into `LogError`, e.g.:

```
DebugLogger.LogError(unexpectedStatusCode.ToString(), e);
```

and

```
public static void LogError(string details, Exception? err)
{
    if (null != logger)
    {
        logger.LogError(err, details);
    }
    else if (null != Exception)
    {
        System.Console.WriteLine(details + Environment.NewLine + err.Message + Environment.NewLine + err.StackTrace);
    }
    else
    {
        System.Console.WriteLine(details);
    }
}
```

Reply:
Fair point.